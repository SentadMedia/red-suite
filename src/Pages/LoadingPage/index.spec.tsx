import LoadingPageWithTheme, { LoadingPage } from '.';

import React from 'react';
import { shallow } from 'enzyme';

describe('LoadingPage', () => {
  const theme = { palette: { primary: { main: '#24292e' } } };
  it('Renders link to Google', () => {
    const loadingPage = shallow(<LoadingPage theme={theme} />);
    expect(loadingPage).toMatchSnapshot();
  });

  it('Renders link to Google with classname', () => {
    const loadingPage = shallow(<LoadingPage theme={theme} />);
    expect(loadingPage).toMatchSnapshot();
  });
});
