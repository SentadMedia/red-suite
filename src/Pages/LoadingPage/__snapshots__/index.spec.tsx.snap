// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage Renders link to Google 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  data-test="Souf"
  direction="column"
  justify="center"
  spacing={0}
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    data-test="innerGrid"
    item={true}
    xs={3}
  >
    <Spinner
      color="#24292e"
      data-test="spinner"
      fadeIn="full"
      name="ball-grid-beat"
      noFadeIn={false}
      overrideSpinnerClassName=""
    />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;

exports[`LoadingPage Renders link to Google with classname 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  data-test="Souf"
  direction="column"
  justify="center"
  spacing={0}
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    data-test="innerGrid"
    item={true}
    xs={3}
  >
    <Spinner
      color="#24292e"
      data-test="spinner"
      fadeIn="full"
      name="ball-grid-beat"
      noFadeIn={false}
      overrideSpinnerClassName=""
    />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
